### DISCUSSION TITLE
Is there a way to control the direction of a course that is being generated

### DISCUSSION THREAD

**Discussion by DaveBrune** [Courseplay/Courseplay_FS22#2543] [Q&A] [answered]
So using the auto generate course in course play, what I'm having issues with is it generates a course for fields east to west, instead of what I want which is north to south.  The reason I want north to south is because going east to west the combine is running into trees which I either have to cut down and remove, or stop course play, take over manually, then restart course play.

Anyone have any suggestions?  I just can't seem to figure out how to tell it which direction to use when generating a course.

**Comment by nitroSkirata** [comment]
There sure is, its conveniantly called "Row direction" in the course generator

(might require the activation of Expert Mode)
![image_2023-05-20_220557427](https://github.com/Courseplay/Courseplay_FS22/assets/9157835/ad6d5106-0a56-45b9-8e46-9cfc2e92e9b2)

**Reply by DaveBrune** [reply, marked as answer]
That did it, I did not see those settings.  Thanks so much!
